commerce_coordinator/apps/frontend_app_payment/pipeline.py

```diff
@@ -0,0 +1,23 @@
+"""
+Legacy filter pipeline implementation
+"""
+
+from django.conf import settings
+from openedx_filters import PipelineStep
+
+from commerce_coordinator.apps.core.constants import PipelineCommand
+from commerce_coordinator.apps.frontend_app_payment.constants import FRONTEND_APP_PAYMENT_CHECKOUT
+
+
+class GetLegacyRedirectUrl(PipelineStep):
```

Comment:
Can we call this `GetPaymentMFERedirect`, [as spec'ed](https://github.com/edx/commerce-coordinator-private/pull/83)? "Legacy" here may be unclear down the line, when other systems may be legacy.